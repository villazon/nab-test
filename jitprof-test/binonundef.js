var a = "1";
var b;
var c = a+b;
b = "2";
c = a+b;
