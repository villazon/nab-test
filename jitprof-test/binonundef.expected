---------------------------
Report of binary operation on undefined value:
 * [location: (jitprof-test/binonundef.js:3:9:3:12)] <- No. usages: 1
Number of statements that perform binary operation on undefined values: 1
[****]BinaryOpUndef: 1
